domains/2giosangmitom.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "2giosangmitom",
+        "email": "yeuxacucodon.dev@proton.me"
+    },
+    "record": {
+        "CNAME": "2giosangmitom.github.io"
```

Comment:
domain redirects to is-a.dev

Reply:
I removed custom domain from GitHub pages setting, check it again please.